droid-hal-device.inc

```diff
@@ -425,6 +425,10 @@ sed -i -e '/^[[:space:]]*mount[[:space:]]/s/^/# Removed during droid-hal-device
 # remove the creation of /tmp to prevent wrong permissons 
 sed -i -e '/^[[:space:]]*mkdir[[:space:]]\/tmp[[:space:]]*/s/^/# Removed during droid-hal-device build : /' $RPM_BUILD_ROOT/*rc
 
+if [ $ANDROID_VERSION_MAJOR -ge "7" ]; then
+mkdir -p $RPM_BUILD_ROOT%{_libexecdir}/droid-hybris/system/etc
+cp -a %{android_root}/out/target/product/%{device}/system/etc/init $RPM_BUILD_ROOT%{_libexecdir}/droid-hybris/system/etc/.
```

Comment:
I meant to do the symlinking when building droid-hal i.e. in this build script, assuming of course we get the symlinking working. This way the symlinks to the needed .rc files are always added and present when droid-hal-init is run. However is the symlinking cannot be fixed then a systemd service might be needed.